pyuvdata/uvdata/mwa_corr_fits.py

```diff
@@ -681,14 +709,11 @@ def read_mwa_corr_fits(
             # these are the indices for the data corresponding to the initial
             # antenna/pol pair
             # generate a mapping index array
-            map_inds = np.zeros((self.Nbls * self.Npols), dtype=np.int32)
+            map_inds = np.zeros((self.Nbls * self.Npols), dtype=np.int32, order="C",)
```

Comment:
Is the `order` keyword needed  for a 1-d array?

Reply:
Probably not? But I just want to ensure the memory will be the way it is expected when using memoryviews as inputs.